lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_build_prechecks.rb

```diff
@@ -3,20 +3,22 @@ module Actions
     class AndroidBuildPrechecksAction < Action
       def self.run(params)
         require_relative '../../helper/android/android_version_helper'
+        require_relative '../../helper/git_helper'
 
         UI.user_error!("Can't build beta and final at the same time!") if params[:final] && params[:beta]
 
         Fastlane::Helper::GitHelper.ensure_on_branch!('release') unless other_action.is_ci()
 
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
```

Comment:
ditto here… and in all other actions where you used that logic of course 😉 
```suggestion
        app = params[:app]
```